src/utils/types.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+export const isStringOrNonEmptyArray = (value: unknown): value is string | unknown[] => {
```

Comment:
non empty either way, I would name it isNonEmptyStringOrArray

Reply:
done